davinci-grid.md

```diff
@@ -30,24 +30,38 @@ to some special commands provided by `ganga`.
 To create your first `ganga` job type the following:
 
 ```python
-j = Job(application=DaVinci(version='v41r2'))
+j = Job(name='First ganga job')
+myApp = prepareGaudiExec('DaVinci','v41r2', myPath='.')
+j.application = myApp
```

Comment:
Right. If we want to be more pythonic (i.e. don't suggest that the user can later modify `myApp` and expect a change in the job), we can just construct the job in one go:
```python
j = Job(name='...', application=myApp, ...)
```
I'm not sure though if this is the best for a tutorial.